src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -527,7 +526,7 @@ private string EncryptToken(string innerJwt, EncryptingCredentials encryptingCre
 
                 try
                 {
-                    var rawHeader = Base64UrlEncoder.Encode(Encoding.UTF8.GetBytes(header.ToString(Newtonsoft.Json.Formatting.None)));
+                    var rawHeader = Base64UrlEncoder.Encode(Encoding.UTF8.GetBytes(header.ToString(Microsoft.IdentityModel.Json.Formatting.None)));
```

Comment:
Quick Q for here and elsewhere:  Do you really need the fully qualified namespace (Microsoft.IdentityModel.Json)?  You have using blocks at the top of the file.  Are you hitting collisions?

Reply:
No collisions, these are remnants of replacing Newtonsoft.Json with Microsoft.IdentityModel.Json. I'll make the change as I don't really need FQN here.